File: sdk/java/src/main/java/com/kucoin/universal/sdk/model/UnifiedWsMessage.java
package com.kucoin.universal.sdk.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import java.util.HashMap;
import java.util.Map;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@Data
@NoArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class UnifiedWsMessage {
  /** User-defined ID (not orderid) is used to uniquely represent a request */
  @JsonProperty("id")
  private String id;

  /** Response data */
  @JsonProperty("data")
  private Map<String, Object> data = new HashMap<>();

  /** Command options */
  @JsonProperty("op")
  private String op;

  /** Gateway error code, 200000 means ok */
  @JsonProperty("code")
  private String code;

  /** Error message */
  @JsonProperty("msg")
  private String message;

  /** Gateway in time(ms) */
  @JsonProperty("inTime")
  private Long inTime;

  /** Gateway out time(ms) */
  @JsonProperty("outTime")
  private Long outTime;

  /** Rate limiting data */
  @JsonProperty("userRateLimit")
  private RateLimit rateLimit;
}

File: sdk/php/src/Model/UnifiedWsMessage.php
<?php

namespace KuCoin\UniversalSDK\Model;

use JMS\Serializer\Annotation\Type;
use JMS\Serializer\Annotation\SerializedName;
/**
 * Class UnifiedWsMessage
 * Represents a message from the unified WebSocket API.
 */
class UnifiedWsMessage
{
    /**
     * User-defined ID (not orderid) is used to uniquely represent a request
     * 
     * @var string
     * @Type("string")
     * @SerializedName("id")
     */
    public $id;

    /**
     * Response data
     * 
     * @var array
     * @Type("array")
     * @SerializedName("data")
     */
    public $data = [];

    /**
     * Command options
     * 
     * @var string
     * @Type("string")
     * @SerializedName("op")
     */
    public $op;

    /**
     * Gateway error code, 200000 means ok
     * 
     * @var string
     * @Type("string")
     * @SerializedName("code")
     */
    public $code;

    /**
     * Error message
     * 
     * @var string
     * @Type("string")
     * @SerializedName("msg")
     */
    public $message;

    /**
     * Gateway in time(ms)
     * 
     * @var int
     * @Type("int")
     * @SerializedName("inTime")
     */
    public $inTime;

    /**
     * Gateway out time(ms)
     * 
     * @var int
     * @Type("int")
     * @SerializedName("outTime")
     */
    public $outTime;

    /**
     * Rate limiting data
     * 
     * @var RateLimit
     * @Type("KuCoin\UniversalSDK\Model\RateLimit")
     * @SerializedName("userRateLimit")
     */
    public $rateLimit;
}